src/database/seeders/20191114145912-users.js

```diff
@@ -6,11 +6,32 @@ module.exports = {
         firstName: 'John',
```

Comment:
I see this file and the dem-users are almost related, maybe you can do away with one.